lib/resty/etcd/v3.lua

```diff
@@ -446,8 +451,10 @@ local function request_chunk(self, method, host, port, path, opts, timeout)
 
         if body.result.events then
             for _, event in ipairs(body.result.events) do
-                event.kv.value = decode_base64(event.kv.value)
-                event.kv.value = decode_json(event.kv.value)
+                if event.type ~= OP_ENUM.DEL then   -- DELETE not have value
```

Comment:
`if event.kv.value then`

how about we check the `value` first?

Reply:
fix it